apps/jobs/views.py

```diff
@@ -538,66 +542,117 @@ def get_submission_by_pk(request, submission_id):
 })
 @throttle_classes([UserRateThrottle,])
 @api_view(['PUT',])
-@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail, IsChallengeCreator,))
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail,))
 @authentication_classes((ExpiringTokenAuthentication,))
 def update_submission(request, challenge_pk):
     """
     API endpoint to update submission realted attributes
+
+    Query Parameters:
+
+     - ``challenge_phase``: challenge phase id, e.g. 123 (**required**)
+     - ``submission``: submission id, e.g. 123 (**required**)
+     - ``stdout``: Stdout after evaluation, e.g. "Evaluation completed in 2 minutes" (**required**)
+     - ``stderr``: Stderr after evaluation, e.g. "Failed due to incorrect file format" (**required**)
+     - ``submission_status``: Status of submission after evaluation
+                (can take one of the following values: FINISHED/CANCELLED/FAILED), e.g. FINISHED (**required**)
+     - ``result``: contains accuracies for each metric, e.g. 
+                [
+                    {
+                        "split": "split1",
```

Comment:
One more thing, I forgot to mention that can we please change it like `"split": "split1 codename"`. It would be accurate and will avoid confusion.

Reply:
@deshraj What about this one? Can I go ahead and update it?

Reply:
Yeah, that's a good idea. I will do these changes.